pages/dr9/sga.rst

```diff
@@ -10,178 +10,866 @@
+  coordinates placed it *inside* the elliptical aperture of *another* (non-SDSS
+  and non-2MASS) galaxy with diameter greater than 0.5 arcmin. Based on visual
+  inspection, we found that many of these sources were due to shredding or were
+  spurious sources with grossly over-estimated diameters.
+
+In addition, we visually inspected all galaxies in the sample with
+:math:`D(25)>0.75\ \mathrm{arcmin}`, including all the NGC/IC galaxies, and
+assessed their published elliptical geometry and coordinates. Where necessary,
+we updated the diameter, position angle, minor-to-major axis ratio, and, in some
+cases, central coordinates "by hand", as indicated in the `BYHAND` column
+described in the data model below. The `NASA Extragalactic Database (NED)`_
+proved invaluable for these cross-checks.
+
+``Supplemental Catalogs``
+-------------------------
+
+To improve the completeness of the `Hyperleda`_ catalog, we supplemented the
+sample with several additional catalogs:
+
+* First, we added the sample of Local Group Dwarf Galaxies from `McConnachie
+  (2012)`_, making sure to account for any systems already in the `Hyperleda`_
+  catalog. Using visual inspection, we determined that approximately half these
+  systems were too unresolved to be part of the ``SGA-2020`` (e.g., Ursa Minor),
+  and so we removed them from the sample.
+  
+..  
+  
+
+* Next, we identified the sample of galaxies in the `RC3`_ and `OpenNGC`_
+  catalogs which were missing from the `Hyperleda`_ sample. Surprisingly, many
+  of these systems are large and high surface-brightness.
+  
+..  
+  
+
+* Finally, we used the `DR8`_ photometric catalogs to identify additional additional large-diameter
+  galaxies. This supplemental catalog consists of two subsamples:
+
+  * First, after applying a variety of catalog-level quality cuts (and extensive
+    visual inspection), we identified all objects in `DR8`_ with half-light radii
+    :math:`r(50)>14\ \mathrm{arcsec}` based on their ``Tractor`` model fits;
+
+  * And second, we constructed a candidate sample of compact galaxies which
+    would otherwise be forced to be point sources in `DR9`_ based on their Gaia
+    catalog properties `this notebook`_ for details.
+
+..  
+  In addition, Fornax and Sculptor to the [http://link/to/the/cluster/page
+  "globular cluster"] sample for special handling in source detection and
+  photometry.
+
+``Group Catalog``
+-----------------
+
+Galaxies which are close to one another (in apparent, angular coordinates) must
+be analyzed jointly. Consequently, we built a simple group catalog using a
+friends-of-friends algorithm and a :math:`10\ \mathrm{arcmin}` linking length,
+taking care to ensure that galaxies which overlap (within two times their
+circularized :math:`D(25)` diameter) were assigned to the same group.
+
+Using this procedure, we identified :math:`\approx515,000` unique groups, of
+which approximately :math:`93\%` have just one member. Among the remaining 7% of
```

Comment:
You really don't need math mode here.  Just write `93%`.  You have `7%` in the same line!